src/altinn-app-frontend/.prettierrc.json

```diff
@@ -0,0 +1,3 @@
+{
+  "singleAttributePerLine": true
```

Comment:
@olemartinorg I missed this during review, there is a prettier config on root which is the one that should be used. Now we have a separate config for altinn-app-frontend subfolder which only has this rule (and prettier default for rest) and other rules (root folder config) for all other files...

Reply:
Fixed in https://github.com/Altinn/app-frontend-react/pull/302 🙂